azkaban-exec-server/src/main/java/azkaban/execapp/AzkabanExecutorServer.java

```diff
@@ -339,28 +340,16 @@ public static void main(String[] args) throws Exception {
       return;
     }
 
-
     /* Initialize Guice Injector */
-    SERVICE_PROVIDER.setInjector(Guice.createInjector(
-        new AzkabanCommonModule(props),
-        new AzkabanExecServerModule()
-    ));
+    final Injector injector = Guice.createInjector(new AzkabanCommonModule(props), new AzkabanExecServerModule());
+    SERVICE_PROVIDER.setInjector(injector);
 
-    launch(props);
+    launch(injector.getInstance(AzkabanExecutorServer.class));
```

Comment:
Just curious what is the intention/benefit of launching from server instance instead of props here?